CorsixTH/Lua/config_finder.lua

```diff
@@ -413,10 +413,8 @@ new_graphics_folder = nil -- [[X:\ThemeHospital\Graphics]]
 screenshots = nil -- [[X:\ThemeHospital\Screenshots]]
 
 -------------------------------------------------------------------------------------------------------------------------
--- High quality (MP3 rather than MIDI) audio replacements.
--- If you want to listen to high quality MP3 audio rather than the original XMI
--- (MIDI) audio, then follow these steps:
---  1) Find MP3 versions of the original tracks (for example the remixes by ZR
+-- If you want to listen to non-Theme-Hospital music, then follow these steps:
+--  1) Find updated versions of the original tracks (for example the remixes by ZR
 --     from http://www.a-base.dds.nl/temp/ThemeHospital_ZRRemix.zip ) or any
 --     other music you want to listen to.
 --  2) Ensure that SMPEG.dll (or equivalent for your platform) is present.
```

Comment:
`SMPEG.dll` is not used on any platform anymore, and different libraries are needed for different sound formats. This comment should be generalized.

Reply:
Actually remove that line completely. This is for end users, and we will ship with all relevant libraries.